src/Sylius/Behat/Context/Ui/Shop/ThemeContext.php

```diff
@@ -11,7 +11,7 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Behat\Context\Ui;
+namespace Sylius\Behat\Context\Ui\Shop;
 
 use Behat\Behat\Context\Context;
 use Sylius\Behat\Page\Admin\Channel\IndexPageInterface;
```

Comment:
I think we should put a little bit more effort in this split. These contexts have a dependency to the both shop and admin. Maybe we can just split them?

Reply:
@lchrusciel Seems greet, I totally agree.